Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Elemental Lords religions update

## Changelog:
- Updated the localization of elemental lords religions, and small reorganization of files.
- Added a new holy order: Aquelian Honor Guards.
- The name of every religion and faiths has changed in the files to avoid confusion in the future.
- Updated also the doctrines.


## Tests:


# How to test:
- Open the game, and check error.log and changed files
- A lot of names has changed in the files so, please, check error.log
